src/pint/event_toas.py

```diff
@@ -122,7 +122,14 @@ def _get_timeref(hdu):
 
 
 def load_fits_TOAs(
-    eventname, mission, weights=None, extension=None, timesys=None, timeref=None
+    eventname,
+    mission,
+    weights=None,
+    extension=None,
+    timesys=None,
+    timeref=None,
+    minmjd=0,
```

Comment:
I suppose these exist to save loading time? (one could always select down the resulting TOAs) Might be worth mentioning in the docstring?

Reply:
Yes, I moved the selection upstream to avoid instantiating TOA object we don't need -- it's very, very slow compared to an array operation.  I could add something to the docstring.